.github/workflows/realm-generator.yml

```diff
@@ -14,32 +14,46 @@ on:
       - .github/workflows/realm-generator.yml
 
 jobs:
-  Generator_CI:
-    runs-on: ${{ matrix.os }}
-
-    strategy:
-      matrix:
-        os: [ubuntu-latest, macos-latest] # windows is not tested ue to bug in how error messages are rendered
-        channel: [stable, beta]
-
-    defaults:
-      run:
-        working-directory: ./generator/
+  CI:
+    runs-on: ubuntu-latest
+    env:
+      REALM_CI: true
 
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
 
       - name : Setup Dart SDK
         uses: dart-lang/setup-dart@main
         with:
-          sdk: ${{ matrix.channel }}
+          sdk: stable
 
-      - name: Install dependencies
+      - name: Install generator dependencies
         run: dart pub get
+        working-directory: ./generator/
```

Comment:
Isn't it correct first to set working directory to `./generator/` and then to do `dart pub get`

Reply:
I am not sure what you mean.  This does both. There is not first and second run.

Reply:
I understand. Order doesn't matter.